azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/implementation/NetworkPeeringsImpl.java

```diff
@@ -82,7 +82,7 @@ public Completable deleteByParentAsync(String groupName, String parentName, fina
                 @Override
                 public Observable<NetworkPeering> call(NetworkPeering localPeering) {
                     if (localPeering == null) {
-                        return Observable.just(localPeering);
+                        return Observable.just(null);
```

Comment:
[P3]: Behavior of `Observable.just(localPeering)` and `Observable.just(null)` are same in this context. Agree that for code readability we should use `Observable.just(null)`.